### DISCUSSION TITLE
v2ray +pihole+tls+ws

### DISCUSSION THREAD

**Discussion by silvermoon1400** [v2fly/v2ray-core#2520] [General]
Newly, the block based on the SNI white list has been activated in Iran, this has made it more difficult to access the Internet, and even most of the large IP data centers are filtered for Iran.
In Iran, methods without TLS work better now, which have low security
Can using private dns in v2ray that I chose pihole here be a solution?
My main idea is to create a fake ptr record that my server IP points to a famous Iranian site to fool the firewall.

**Comment by nursery01** [comment]
> My main idea is to create a fake ptr record that my server IP points to a famous Iranian site to fool the firewall.

You mean to modify ptr to point to a legitimate sni, and make a legitimate sni to point to your ip

I think it should be ok, if isp do not have an IP whitelist mechanism

**Reply by silvermoon1400** [reply]
> پاسخ
IP white list is active in Iran
But white IP is still found
